tests/foreman/maintain/test_backup_restore.py

```diff
@@ -452,13 +452,14 @@ def test_positive_backup_restore(
 
     expected_files = get_exp_files(sat_maintain, skip_pulp)
     assert set(files).issuperset(expected_files), assert_msg
-
+    config_files = sat_maintain.execute(
+        f"tar -tf {backup_dir}/config_files.tar.gz"
+    ).stdout.splitlines()
     # Check if certificate tar file is present in Capsule backup.
     if instance == 'capsule':
-        cert_file = sat_maintain.execute(
-            f'tar -tvf {backup_dir}/config_files.tar.gz | grep {sat_maintain.hostname}'
-        ).stdout
-        assert f'{sat_maintain.hostname}-certs.tar' in cert_file
+        assert f'root/{sat_maintain.hostname}-certs.tar' in config_files
+    # Check Ansible configuration file under /etc/ansible is present
```

Comment:
Isn't the Python line self describing? I noticed that robottelo has that a lot and I'm trying to understand why. To me it just adds bloat to the code.

Reply:
yeah, right. Thanks for pointing it out, I slowly started to fall into that trap as well😄